sunpy/instr/goes.py

```diff
@@ -69,8 +69,11 @@
 from sunpy.util.net import check_download_file
 from sunpy import sun
 
+GOES_CONVERSION_DICT = {-4: 'X', -5: 'M', -6: 'C', -7: 'B', -8: 'A'}
```

Comment:
sincerely I think hte dictionary makes more sense the other way around, class: value.  This way, if I see:
```python
GOES_CONVERSION_DICT[-4]
```
I don't confuse it with a list from where the last 4th element is being taken.
Also, instead of -4, it could include the factor (1e-4) and even the unit.